engine/runtime-instrument-common/src/main/java/org/enso/interpreter/instrument/job/VisualizationResult.java

```diff
@@ -43,6 +43,14 @@ public static String findExceptionMessage(Throwable ex) {
     }
   }
 
+  public static boolean isInterruptedExceptionValue(Object object) {
+    if (object instanceof Throwable ex) {
+      return isInterruptedException(ex);
+    } else {
+      return false;
+    }
+  }
+
```

Comment:
Is it necessary to introduce new `isInterruptedExceptionValue` method? Isn't it easier to change the argument of existing `isInterruptedException` to `Object`?

Reply:
Instance check didn't feel right, hence another method. But I don't have a preference tbh.

Reply:
There is no need to perform an instance check. `isInterruptedException(Object ex, InteropLibrary iop)` accepts `Object`.